flake8 is failing

```
Run flake8 .
./ibis_bigquery/udf/__init__.py:19:16: E203 whitespace before ':'
./ibis_bigquery/udf/__init__.py:19:80: E501 line too long (85 > 79 characters)
./tests/unit/test_client.py:10:80: E501 line too long (81 > 79 characters)
./tests/udf/test_core.py:6:1: F401 'ibis.compat.PY38' imported but unused
./tests/udf/test_udf_execute.py:8:1: F401 'ibis.compat.PY38' imported but unused
./tests/udf/test_find.py:3:1: F401 'pytest' imported but unused
./tests/udf/test_find.py:4:1: F401 'ibis.compat.PY38' imported but unused
Error: Process completed with exit code 1.
```

--- a/ibis_bigquery/udf/__init__.py
+++ b/ibis_bigquery/udf/__init__.py
@@ -16,7 +16,9 @@ from ..udf.core import PythonToJavaScriptTranslator
 
 __all__ = ('udf',)
 
-_udf_name_cache : Dict[str, Iterable[int]] = collections.defaultdict(itertools.count)
+_udf_name_cache: Dict[str, Iterable[int]] = (
+    collections.defaultdict(itertools.count)
+)
 
 
 def create_udf_node(name, fields):

--- a/tests/udf/test_core.py
+++ b/tests/udf/test_core.py
@@ -3,7 +3,6 @@ import sys
 import tempfile
 
 import pytest
-from ibis.compat import PY38
 
 from ibis_bigquery.udf.core import PythonToJavaScriptTranslator, SymbolTable
 

--- a/tests/udf/test_find.py
+++ b/tests/udf/test_find.py
@@ -1,7 +1,5 @@
 import ast
 
-import pytest
-from ibis.compat import PY38
 from ibis.util import is_iterable
 
 from ibis_bigquery.udf.find import find_names

--- a/tests/udf/test_udf_execute.py
+++ b/tests/udf/test_udf_execute.py
@@ -5,7 +5,6 @@ import ibis.expr.datatypes as dt
 import pandas as pd
 import pandas.testing as tm
 import pytest
-from ibis.compat import PY38
 from pytest import param
 
 import ibis_bigquery

--- a/tests/unit/test_client.py
+++ b/tests/unit/test_client.py
@@ -7,7 +7,11 @@ import ibis_bigquery.client
     ["project", "dataset", "expected"],
     [
         ("my-project", None, ("my-project", "my-project", None)),
-        ("my-project", "my_dataset", ("my-project", "my-project", "my_dataset")),
+        (
+            "my-project",
+            "my_dataset",
+            ("my-project", "my-project", "my_dataset"),
+        ),
         (
             "billing-project",
             "data-project.my_dataset",

